CONTRIBUTING.rst

```diff
@@ -122,19 +152,24 @@ Dependencies for production Pontoon are in ``requirements.txt``. Development dep
 
 Note that we use a specific format for our dependencies, in order to make them more maintainable. When adding a new requirement, you should add it to the appropriate section and group it with its sub-dependencies if applicable.
 
-For example, to add ``foobar`` version 5::
+For example, to add ``foobar`` version 5:
```

Comment:
What's the deal with replacing all `::` with `:`?

Reply:
That's `.rst` for you. If you end a line with `::` then it considers the next one as a block of code. Thus, because I added all those `.. code-block:: shell`, I had to remove the colons, otherwise it would turn `.. code-block:: shell` into a code block and show the actual code as regular text.